.github/workflows/ci.yaml

```diff
@@ -41,6 +41,8 @@ jobs:
         sudo snap install juju-wait --classic
         # https://github.com/charmed-kubernetes/actions-operator/issues/27
         sudo snap refresh charmcraft --channel latest/candidate
+        sudo apt-get install python3-setuptools
+        sudo pip3 install tox
 
     - name: Build and deploy charm
       run: |
```

Comment:
This block (I think - correct me if I'm wrong!) is what you're now recreating in pytest, so it can be removed